intellij-ext/src/main/kotlin/com/metalbear/mirrord/MirrordListener.kt

```diff
@@ -116,9 +118,11 @@ class MirrordListener : ExecutionListener {
                         mirrordEnv["MIRRORD_FILE_OPS"] = fileOps.isSelected.toString()
                         mirrordEnv["MIRRORD_AGENT_TCP_STEAL_TRAFFIC"] = stealTraffic.isSelected.toString()
                         mirrordEnv["MIRRORD_EPHEMERAL_CONTAINER"] = ephemeralContainer.isSelected.toString()
-                        mirrordEnv["MIRRORD_REMOTE_DNS"] = remoteDns.isSelected.toString()
                         mirrordEnv["MIRRORD_TCP_OUTGOING"] = tcpOutgoingTraffic.isSelected.toString()
                         mirrordEnv["MIRRORD_UDP_OUTGOING"] = udpOutgoingTraffic.isSelected.toString()
+                        if(tcpOutgoingTraffic.isSelected || udpOutgoingTraffic.isSelected) {
+                            mirrordEnv["MIRRORD_REMOTE_DNS"] = "true"
+                        }
```

Comment:
we want to keep this as close to the CLI and VSCode extension, where we keep remote DNS as a toggle option, so it'd best to keep it to the user to decide if they want to use this

Reply:
Ok, I will change this.